CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Draft
-
+- Replace Twitter logo with X logo within social sharing and social link components [#2386]
```

Comment:
@AndrewBarber Could you add a link after PR's number please
`(https://github.com/bigcommerce/cornerstone/pull/2387)`
and fix please number of PR
`2386 -> 2387`

Reply:
Hey, sorry I don't understand what you're asking me here.